### PR TITLE
Dev/m.jankowski/implement lkvm handler

### PR DESCRIPTION
**Pull request by MJJSoftware**

### CODE REVIEW

File: warden/warden_daemon/src/daemon.rs [hunk 1]
```diff
@@ -49,7 +52,19 @@ impl Daemon {
...
+                    if cli.cca_enable {
+                        runner.configure_cca_settings();
+                    }
+                    Ok(Box::new(VmRunner::new(runner, path)))
+                }
+                false => {
+                    let runner = QemuRunner::new(cli.virtualizer_path.clone(), config);
+                    Ok(Box::new(VmRunner::new(runner, path)))
+                }
+            }),
```
Comment (L0czek) [inline warden/warden_daemon/src/daemon.rs:67]: Why not use a simple If when matching boolean?

File: warden/warden_daemon/src/virtualization/vm_runner/lkvm.rs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        runner.setup_kernel(&config.kernel);
+        runner.setup_memory(&config.memory);
+        runner.setup_network(&config.network);
+        runner.control_output();
+        runner
+    }
+    pub fn configure_cca_settings(&mut self) {
+        self.command.arg("--debug");
+        self.command.arg("--irqchip=gicv3");
+        self.command.arg("--disable-sve");
```
Comment (L0czek) [inline warden/warden_daemon/src/virtualization/vm_runner/lkvm.rs:29]: minor: These two aren't exactly related to CCA. The first selects the interrupt controller and the second one disables some ARM vector extension.

**Review by L0czek — APPROVED** [review]
LGTM, minor stuff

File: warden/warden_daemon/src/virtualization/vm_runner/lkvm.rs (hunk 2 above)
Comment (MJJSoftware) [reply warden/warden_daemon/src/virtualization/vm_runner/lkvm.rs]: Done

**Pull request merged**